Jenkinsfile_k8s

```diff
@@ -1,5 +1,6 @@
-
+def isPrimary = false
```

Comment:
```suggestion
def isBranchPrimary = false
```

WDYt about this name proposal? Rationale is that the qualificative "primary" could be confusing. By adding the name "branch", it aims to make explic when reading the name that "branch" is the subject of the qualificative

Reply:
I think I mistakenly clicked a button around here ... sorry ....

Reply:
@smerle33 which button? I don't see any kind of modification, should be fine!